components/SearchBar/SearchBar.tsx

```diff
@@ -66,6 +67,7 @@ const SearchBar = ({
   useOnClickOutside(dropdownRef, handleClickOutside);
 
   useEffect(() => {
+    !isSearchInputFocusable && setIsSearchInputFocusable(!isSearchInputFocusable);
```

Comment:
So here the "SeachInputFocusable" state is being toggled, but what does that do or affect? As far as I can see  it's not being read anywhere.

Reply:
Yea I don't need it.